_posts/2019-08-11-pre-post-events-sustainable-train-travels.md

```diff
@@ -0,0 +1,152 @@
+So you have the flexibility to ride any train outside of your home country on 7 days.
+In addition, the ticket is also valid for travel in your country of residence during one outbound journey and one inbound journey that occur during travel days at any point in your trip.
+
+Most high-speed and night trains require a [reservation at an additional cost](https://www.interrail.eu/en/plan-your-trip/reservations).
+
+For most European citizens [the Interrail pass (currently 10% off)](https://www.interrail.eu/en/autumn-sale/global-pass) is probably the most flexible and affordable ticket.
+For [Non-EWR citizens](https://www.interrail.eu/en/help/ordering-pass/do-i-need-interrail-or-eurail-pass) there is [Eurail](https://www.eurail.com/en/eurail-passes/global-pass).
+
+### Tools
+
+Within Spain, Portugal, France:
+* [Trenes](https://train.trenes.com/) (Don't forget to uncheck "Also search for accomodation")
+
+Central Europa:
+* [SNCF](https://www.sncf.com/en)
+
+For Northern Europa:
+* Austria, Germany, Czech Republic, Slovakia, Hungary, Italy, Switzerland, Serbia -> [ÖBB](https://www.oebb.at/)
+* Sweden, Denmark, Netherlands, Germany, Switzerland -> [Deutsche Bahn](https://bahn.de)
+* Switzerland, France, Spain -> [Schweizerische Bundesbahnen](https://sbb.ch)
+
+## The travel to Barcelona
+
+Now let us look at some examples:
+
+### Within Spain
+
+From Madrid to Barcelona there are two kinds of trains traveling.
+
+* A cheaper train that takes 9.5 hours.
+* With the high speed train you can reach Barcelona in 2.5 hours.
+
+* From Sevilla Santa Justa you travel directly to Barcelona within 5.5 hours.
+
+### From France or Portugal
+
+From most bigger cities of France you can reach Barcelona within 7 to 9 hours.
+
+* From Paris, there is a direct TGV connection 3 times a day.
+* From Rennes, take the TGV via Nimes.
+* From Bordeaux, with InterCity and TGV via Narbonne.
+
+* From Lisboa Estacion Oriente there is a train to Madrid Chamartin. Since it is not a high speed train it takes 11 hours and you will probably need to continue on the second day.
+
+Note that there is a [cheaper Interrail country pass](https://www.interrail.eu/en/interrail-passes/one-country-pass/spain-passes) and you probably only need 3 days within 1 month.
+
+
+### From the North
+
+These events are timed so that people traveling towards Barcelona can stop by.
+What does that mean, let's go through a possible option:
+
+{:.layout-auto}
+|              City               |    Day    | Departure | Arrival |                                           Notes                                           |
+| ------------------------------- | --------- | --------- | ------- | ----------------------------------------------------------------------------------------- |
+| Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                          |
```

Comment:
I still find the "by the office" thing confusing. Make it "your office", "favorite coffee shop" or something, making it clear we don't have any on-site event planned.

Reply:
fixed